tests/benchmarks/cli/commands/test_exp_run.py

```diff
@@ -0,0 +1,4 @@
+def test_exp_run(bench_dvc, tmp_dir, scm, dvc, make_dataset, pipeline):
+    make_dataset(files=True, dvcfile=True, cache=True)
+    bench_dvc("exp", "run")
+    bench_dvc("exp", "run", name="noop")
```

Comment:
It feels strange to configure by hand the dataset, while using an implicit hardcoded pipeline. As a dvc-bench user, I would much like something like:

```suggestion
def test_exp_run(bench_dvc, project):
    bench_dvc("exp", "run")
    bench_dvc("exp", "run", name="noop")
```

esp since the commands are so generic.

If a test wants a specific project to be tested on, we could create a fixture for it and use istead of `project`. E.g. `lstm_seq2seq` for https://github.com/dberenbaum/lstm_seq2seq/tree/dvc